src/components/ArticleList.jsx

```diff
@@ -0,0 +1,67 @@
+import { useEffect, useState } from 'react';
+import { fetchArticles } from '../utils';
+
+export default function ArticleList() {
+  const [articleList, setArticleList] = useState([]);
+
```

Comment:
Could we have some state to track the loading status of our fetch, feeds back to the user about the progress of the asynchronous call.